clang/test/Driver/sycl-offload-new-driver.c

```diff
@@ -32,7 +32,7 @@
 
 /// Verify options passed to clang-linker-wrapper
 // RUN: %clangxx --target=x86_64-unknown-linux-gnu -fsycl --offload-new-driver \
-// RUN:          --sysroot=%S/Inputs/SYCL -### %s 2>&1 \
+// RUN:          -fsycl-instrument-device-code --sysroot=%S/Inputs/SYCL -### %s 2>&1 \
```

Comment:
In this test, it would be better to remove the device libraries in the `clang-linker-wrapper` call instead of adding the option.  Adds a bit more variety on testing.